src/helpers/getDestinationData.js

```diff
@@ -1,9 +1,18 @@
+import { getDestination } from '../redux/Destinations/destinations';
 // import axios from 'axios';
 // const url = 'http://localhost:3000/';
-
+const destinationUrl = 'http://localhost:3000/api/v1/destinations';
 // export const getDestinations = () => async () => {
 //     axios.get(url).then((response) => {
 //     const { data } = response;
 //     return data
 //   });
 // };
+
+const getDestinationThunk = () => async (dispatch) => {
+  const request = await fetch(destinationUrl);
+  const data = await request.json();
+  dispatch(getDestination(data));
+};
+
+export default getDestinationThunk;
```

Comment:
I didn't mean that you should move the whole thunk function to the helper folder. I thought that you should just extract this part of it where you are sending the request:
![image](https://user-images.githubusercontent.com/63932912/142402425-9b1b14f7-9ea6-47f9-95f1-2b6f4f9b3720.png)